src-java/nbworker-topology/nbworker-storm-topology/src/main/java/org/openkilda/wfm/topology/nbworker/bolts/SwitchOperationsBolt.java

```diff
@@ -323,4 +324,9 @@ public void declareOutputFields(OutputFieldsDeclarer declarer) {
         declarer.declareStream(StreamType.TO_SERVER42.toString(),
                 new Fields(MessageEncoder.FIELD_ID_PAYLOAD, MessageEncoder.FIELD_ID_CONTEXT));
     }
+
+    @Override
+    public void islBfdPropertiesChanged(Endpoint source, Endpoint destination) {
+        log.warn("Discard islBfdPropertiesChanged link carrier call");
```

Comment:
Didn't get it. Why we need to discard this call?

Reply:
... this is the ugly design of `nbworker`.

Method `islBfdPropertiesChanged` is part of `ILinkOperationsServiceCarrier`, this interface are implemented by `LinkOperationsBolt` and `SwitchOperationsBolt`. `LinkOperationsBolt` can and must provide a correct implementation of this method, while `SwitchOperationsBolt` can't do it. I have added this warning to have visibility of the issue, when something will goest wrong.